content/intro-to-storybook/svelte/es/using-addons.md

```diff
@@ -0,0 +1,188 @@
+
+Usemos el tipo de knob de objeto en el componente `Task`.
+
+Primero, importe el decorador `withKnobs` y el tipo de knob `object` a `Task.stories.js`:
+
+```javascript
+// src/components/Task.stories.js
+import { action } from "@storybook/addon-actions";
+import { withKnobs, object } from '@storybook/addon-knobs';
+```
+
+A continuación, dentro de la exportación `default` del archivo`Task.stories`, agregue `withKnobs` como elemento de `decorators`:
+
+```javascript
+// src/components/Task.stories.js
+
+export default {
+  title: 'Task',
+  decorators: [withKnobs],
+  // same as before
+};
+```
+
+Por último, integre el tipo de knob `object` dentro de la historia "predeterminada":
+
+```javascript
+// src/components/Task.stories.js
+export const Default = () => ({
+  Component: Task,
+  props: {
+    task: object("task", { ...taskData })
+  },
+  on: {
+    ...actionsData
+  }
+});
+export const Pinned = () => ({
+  Component: Task,
+  props: {
+    task: {
+      ...taskData,
+      state: "TASK_PINNED"
+    }
+  },
+  on: {
+    ...actionsData
+  }
+});
+export const Archived = () => ({
+  Component: Task,
+  props: {
+    task: {
+      ...taskData,
+      state: "TASK_ARCHIVED"
+    }
+  },
+  on: {
+    ...actionsData
+  }
+});
+```
+
+Ahora debería aparecer una nueva pestaña "Knobs" al lado de la pestaña "Action Logger" en el panel inferior.
+
+Como se documenta [aquí](https://github.com/storybooks/storybook/tree/master/addons/knobs#object), el tipo `object` del knob acepta una etiqueta y un objeto predeterminado como parámetros. La etiqueta es constante y se muestra a la izquierda de un campo de texto en el panel de complementos. El objeto que ha pasado se representará como un blob JSON editable. ¡Siempre que envíe un JSON válido, su componente se ajustará en función de los datos que se pasan al objeto!
+
+## Complementos evolucionan el alcance de tus Storybooks
+
+Su instancia de Storybook no solo sirve como un maravilloso [CDD environment](https://blog.hichroma.com/component-driven-development-ce1109d56c8e), sino que ahora estamos proporcionando una fuente interactiva de documentación. Los props son geniales, pero un diseñador o alguien completamente nuevo en el código de un componente podrá descubrir su comportamiento muy rápidamente a través de Storybook con el complemento de knobs implementado.
+
+## Usando Knobs para encontrar casos de borde
+
+Además, con un fácil acceso para editar los datos pasados ​​a un componente, los QA Engineers o los UI Engineers ahora pueden llevar un componente al límite. Como ejemplo, ¿qué le sucede a `Task` si nuestro elemento de la lista tiene un string _GIGANTESCO_?
+
+![¡Oh no! ¡El contenido de la extrema derecha está cortado!](/intro-to-storybook/addon-knobs-demo-edge-case.png) 😥
+
+¡Gracias a poder probar rápidamente diferentes entradas a un componente, podemos encontrar y solucionar estos problemas con relativa facilidad! Arreglemos el problema de desbordamiento agregando un estilo a `Task.js`:
```

Comment:
@IvanSotelo this should be `Task.svelte` not `Task.js`, i'll update the pr i just made #304 with this small update as it "slipped under the radar".